src/main/java/frc/robot/RobotContainer.java

```diff
@@ -490,85 +580,120 @@ private void configureButtonBindings() {
    * @return the command to run in autonomous
    */
   public Command getAutonomousCommand() {
-    return autoChooser.get();
+    if (autoChooser != null) {
+      return autoChooser.get();
+    } else {
+      return Commands.none();
+    }
   }
 
   public Command elv0Command() {
-    return elevator.L0().andThen(waitForElevator().withTimeout(0.5).andThen(elevator.limitHit()));
+    if (elevator != null) {
+      return elevator.L0().andThen(waitForElevator().withTimeout(0.5).andThen(elevator.limitHit()));
+    } else {
+      return Commands.none();
+    }
   }
 
   public Command waitForElevator() {
-    return Commands.waitUntil(elevator.elevatorAtCurrentSetpoint());
+    if (elevator != null) {
+      return Commands.waitUntil(elevator.elevatorAtCurrentSetpoint());
+    } else {
+      return Commands.none();
+    }
   }
 
   public Command intakeAutoCommand() {
-    if (Robot.isSimulation()) {
+    if (Robot.isSimulation() && intake != null) {
       return intake
           .humanPlayerIntake()
           .until(intake.pivotAtSetpoint(kPivotCoralStationAngle))
           .andThen(() -> intake.addGamePieceToIntakeSim())
           .asProxy();
-    } else {
+    } else if (intake != null) {
```

Comment:
Okay as is, but you might consider nesting the original if-else in another if-else checking for the intake != null. Then you don't wind up checking it twice.

```
if (intake != null)  {
  if (Robot.isSimulation()) {
    // return Simulation command
  } else {
    //return Real Command
  }
} else {
  // return Disabled Command
}
```

Same comment for intakeFloorAutoCommand() and scorerAutoCommand() as well